spec/presenters/scope_filters_presenter_spec.rb

```diff
@@ -173,6 +173,20 @@
     end
   end
 
+  describe "#path_id" do
+    include GdsApi::TestHelpers::SupportApi
+
+    it "is true when path_id is present" do
```

Comment:
What does true mean here? I'd expect the test to say `.to be_true` from the above line.

Reply:
I forgot to change the title of the test when I copied over another test, thanks for pointing that out. Will change to check that the path_id is present 👍